public/index.html

```diff
@@ -46,6 +46,11 @@ <h1>体力ゲージ</h1>
       const response = await fetch("/welcome-message");
       document.querySelector("#welcomeMessage").innerText = await response.text();
     };
+
+    window.onload = async (event) => {
+      const response = await fetch("/life-gauge");
+      document.querySelector("lifeGauge").innerText = await response.text();
```

Comment:
```suggestion
      document.querySelector("#lifeGauge").innerText = await response.text();
```

ここに＃から動かないのではないでしょうか？

Reply:
修正してみます。

Reply:
OKです！ありがとうございます。強いていうなら　％も残していただきたいです。

Reply:
了解です